src/snapred/meta/mantid/WorkspaceNameGenerator.py

```diff
@@ -86,7 +85,9 @@ def formatRunNumber(runNumber: str):
     def formatVersion(version: Optional[Version], use_v_prefix: vPrefix = vPrefix.WORKSPACE):
         if version is None:
             return ""
-        if not version == "*":
+        if version != VERSION_DEFAULT:
+            version = str(version).zfill(4)
+        elif isinstance(VERSION_DEFAULT, int):
```

Comment:
No more globs, but there exists a default version which is where the default diffcal table and the initial calibration parameters file is written.  We have been using `v_0000` as the default, but users might want to call this something else, like `default`, and use `v_0000` as the starting version number.